src/SearchResults/index.js

```diff
@@ -400,4 +420,122 @@ SearchResults.prototype.getFacetByName = function(name) {
     find(this.hierarchicalFacets, predicate);
 };
 
+/**
+ * Get the facet values of a specified attribute from a SearchResults object.
+ * @param {SearchResults} results the search results to search in
+ * @param {string} attribute name of the facetted attribute to search for
+ * @return {array|object} facet values enhanced with the attribute facetType to
+ * distinguish the type of facet.
+ */
+function getFacetValues(results, attribute) {
```

Comment:
I wondering which other name we could use here to avoid having the same name twice: `getFacetValues(SearchResults)` VS `SearchResults.prototype.getFacetValues`.